tiledb/sm/array_schema/array_schema.cc

```diff
@@ -153,19 +153,19 @@ ArraySchema::ArraySchema(
 
   // Check array schema is valid.
   st = check_double_delta_compressor(coords_filters_);
-  if (!st.ok())
-    throw StatusException(
-        Status_ArraySchemaError("Array schema check failed; Double delta "
-                                "compression used in zipped coords."));
+  if (!st.ok()) {
```

Comment:
Since we are currently over-writing some of the error messages, I'm inclined to wait until we have support for nested throws with the StatusException and the C-API output.

Reply:
Could we just do check_attribute_dimension_label_names then?